### PR TITLE
solution

### PR DESCRIPTION
**Pull request by pasha1932**
[DEMO LINK](https://pasha1932.github.io/react_todo-app/)

### CODE REVIEW

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done: 
Screenshot 2024-04-23 at 13 52 28
Consider fixing the layout here

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Screenshot 2024-04-23 at 15 36 56
Make sure to deploy your fixes

**Comment by pasha1932** [comment]
[DEMO LINK](https://pasha1932.github.io/react_todo-app/)

**Comment by pasha1932** [comment]
in my project, when i write npm start 
image

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Screenshot 2024-04-23 at 17 10 30
Great work! Consider fixing the editing layout

**Comment by pasha1932** [comment]
image

File: src/components/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  const focusInput = () => {
+    if (inpRef.current) {
+      inpRef.current.focus();
+    }
+  };
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (title.trim() === '') {
```
Comment (sTorba24) [inline src/components/Header.tsx:23]:
```suggestion
    if (!title.trim()) {
```

**Review by sTorba24 — APPROVED** [review]

**Pull request closed without merging**